src/wallet/rpcwallet.cpp

```diff
@@ -1717,7 +1718,7 @@ UniValue listsinceblock(const JSONRPCRequest& request)
             "    \"label\" : \"label\"       (string) A comment for the address/transaction, if any\n"
             "    \"to\": \"...\",            (string) If a comment to is associated with the transaction.\n"
              "  ],\n"
-            "  \"lastblock\": \"lastblockhash\"     (string) The hash of the last block\n"
+            "  \"lastblock\": \"lastblockhash\"     (string) The hash of the block (target_confirmations-1) from the best block on the main chain. This is typically used to feed back into listsinceblock the next time you call it. So you would generally use a target_confirmations of say 6, so you can keep rescanning the last 6 blocks plus any new ones\n"
```

Comment:
"rescanning" means something specific, maybe "so you will be continually re-notified of transactions until they've reached 6 confirmations"